### ISSUE TITLE
Automatically remove `@` symbols?

### ISSUE THREAD

**Issue by Kixunil** [bitcoin-core/bitcoin-maintainer-tools#139]
As discussed in https://github.com/rust-bitcoin/rust-miniscript/pull/450#issuecomment-1253977283 the script currently warns about `@` in PR description but it may be more convenient to jut remove them automatically. Not modifying the text sounds like a good default but having an option to just remove the symbols (possibly only those that really can be mentions) when one sees they are only mentions would be more convenient.

**Comment by okorye** [comment]
{"SPDXID":"SPDXRef-DOCUMENT","spdxVersion":"SPDX-2.3","creationInfo":{"created":"202403-05T10:18:00Z","creators":["Tool: GitHub.com-DependencyGraph"]},"name":"com.github.okorye/codespaces-express","dataLicense":"CC01.0","documentDescribes":["SPDXRef-com.github.okorye-codespacesexpress"],"documentNamespace":"https://github.com/okorye/codespaces-
express/dependency_graph/sbom-ce3fd3ce36630cb4","packages":[{"SPDXID":"SPDXRefcom.github.okorye-codespaces-express","name":"com.github.okorye/codespacesexpress","versionInfo":"","downloadLocation":"git+
https://github.com/okorye/codespaces-express
","filesAnalyzed":false,"supplier":"NOASSERTION","externalRefs":[{"referenceCategory": "PACKAGE{
  "objectType": "merchant",
  "id": "1587",
  "type": "gateway",
  "created": "2022-09-26T13:20:03+00:00",
  "updated": "2022-09-26T13:20:03+00:00",
  "firstName": "Jane",
  "lastName": "Doe",
  "company": "ACME, Inc.",
  "email": "someone@example.com",
  "phone": "555-555-5555",
  "fax": "123-456-7890",
  "url": "",
  "address1": "123 Fake St.",
  "address2": "Unit 101",
  "city": "Chicago",
  "state": "IL",
  "zip": "60601",
  "country": "US",
  "timezone": "America/Chicago",
  "costPlan": null,
  "status": "pending",
  "externalIdentifier": "A Company That Makes Everything",
  "lastTransactionDate": null,
  "parentAffiliateId": "1",
  "accountInfo": {
    "checkAba": "123123123",
    "checkAccount": "*****3123",
    "accountType": "checking",
    "accountHolderType": "business"
  },
  "features": {
    "forceTos": true,
    "forceBillingAuth": true,
    "allowBlindCredits": false,
    "allowBlindCheckCredits": false,
    "advancedTransactionRouting": false,
    "sendBillingEmail": false,
    "braspag": {
      "advancedTransactionRouting": false
    },
    "verifi": {
      "enabled": false
    },
    "viewBillingStatement": false,
    "litle": {
      "reportGroup": false,
      "prePaidIndicator": false,
      "affluentIndicator": false,
      "recycleAdvice": false,
      "campaign": false,
      "affiliate": false,
      "merchantGroupId": false
    },
    "allowEmptyCcExp": false,
    "pastDue": false,
    "apiReceipt": false,
    "batchReceipt": false,
    "delinquent": false,
    "commercialCardIndicator": false,
    "customDescriptor": false,
    "apiAllowOnlyUsername": false,
    "orderIdDuplicateCheck": false,
    "cardLevelResult": false,
    "pmsalem": {
      "ignoreAmexCvv": false,
      "allowNovusVerify": false,
      "idInMdf11": false
    },
    "credorax": {
      "bankNet": false,
      "z5Risk": false
    },
    "vantiv": {
      "remainingBalance": false
    },
    "prosa": {
      "deferredPayment": false
    },
    "cronopay": {
      "customerId": false
    },
    "captureHigher": false,
    "moneris": {
      "orderId": false
    },
    "extendedFileBatchResponse": false,
    "credov3": {
      "ignoreAmexCvv": false
    },
    "enforceRequiredFieldsForSwiped": false,
    "preventDuplicateCvRecords": false,
    "preventConfiguringSurcharge": false,
    "tsys": {
      "paymentFacilitatorFields": false,
      "allowCustomReport": false,
      "alternativePosData": false
    },
    "requireProcessorFieldOnVt": false,
    "isCardTesting": false,
    "allowTipOnVtKeyedTransactions": false,
    "transitForceAcceptance": false,
    "level3": {
      "autoLoadTemplate": false
    },
    "emvPartialApprovals": false,
    "allowOfflineAuths": false,
    "supportsGooglePay": false,
    "supportsApplePay": false,
    "isApplePayEnabled": false,
    "useDeviceRecognition": false
  },
  "needsPendingServicesActivated": true,
  "payFacBillingPeriod": "monthly",
  "rejectedCertificates": "0"
}develop****
Bradenbraidsandmore@gmail.com

**Comment by laanwj** [comment]
Yes i think this makes sense, if you can detect use that is definitely a github username, it could just remove the '@'.  Currently maintainers do this manually.

**Comment by Seefalfares** [comment]
Mohamad Sif Alhamad
